imagersite/imager_profile/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
You made a model, you should register it in this file so it is viewable in the admin.